app/src/main/java/org/metabrainz/mobile/presentation/features/tagger/DirectoryPicker.kt

```diff
@@ -56,10 +55,11 @@ class DirectoryPicker : Fragment(), OnItemCLickListener {
             binding.instructionForTagFix.visibility = View.VISIBLE
             binding.instruction.visibility = View.GONE
             val intent = Intent(Intent.ACTION_OPEN_DOCUMENT_TREE)
-            if (packageManager?.let { it1 -> intent.resolveActivity(it1) } != null) {
+            try {
                 startActivityForResult(intent, REQUEST_CODE_OPEN_DOCUMENT)
-            } else {
-                Toast.makeText(context, "Document provider not found", Toast.LENGTH_SHORT).show()
+            }
+            catch (ex: Exception){
+                Log.e("TaggerDirectoryPicker",ex.message.toString())
```

Comment:
We should add a user friendly toast message so that the user knows something went wrong.